models.go

```diff
@@ -66,12 +66,11 @@ type StagingResult struct {
 	LifecycleMetadata `json:"lifecycle_metadata"`
 	ProcessTypes      `json:"process_types"`
 	ProcessList       []Process `json:"processes, omitempty"`
-	Sidecars          []Sidecars  `json:"sidecars, omitempty"`
-	ExecutionMetadata string      `json:"execution_metadata"`
-	LifecycleType     string      `json:"lifecycle_type"`
+	Sidecars          []Sidecar `json:"sidecars, omitempty"`
```

Comment:
Why is this an array if it only ever has one item in it?

Reply:
I’m not sure I follow, you can have multiple sidecars, each potentially attached to a different process type and with its own command, no?

Reply:
Ah right sorry we reading the field name. LGTM.